base/loading.jl

```diff
@@ -1556,7 +1557,7 @@ function get_uuid_name(project_toml::String, uuid::UUID)
     return get_uuid_name(project, uuid)
 end
 
-function collect_preferences!(project_toml::String, uuid::UUID)
+function collect_preferences(project_toml::String, uuid::UUID)
```

Comment:
This used to be a mutating operation but that got rebased out I suppose.  Good catch!